src/components/InfoSection/InfoSection.css

```diff
@@ -0,0 +1,68 @@
+/** @format */
+
+.infoSection {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  border: 2px solid;
+}
+
+.avatar img {
+  max-width: 14em;
+}
+
+.info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.actionButtons {
+  display: flex;
+  align-items: center;
+  margin-bottom: 1em;
+  width:100%
+}
+
+.links {
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+  display: flex;
+}
+
+.link {
+  color: black;
+  text-decoration: none;
+  padding-right: 15px;
+}
+
+.displayName {
+  align-self: flex-start;
+  display: flex;
+}
+
+.userName {
+  margin-inline-end: 1em;
+  font-size: 1.2em;
+}
+
+.editButton {
+  font-size: .8em;
+  margin-inline-end: 1em;
+  background: white;
+  padding: .7em;
+  cursor: pointer;
+  border: 1px solid #ddd;
+  border-radius: .3em;
+  font-weight: 600;
+}
+
+.settingButton {
+  margin-inline-end: 1em;
+}
+
+.settingButton:link, .settingButton:active, .settingButton:visited, .settingButton:hover {
```

Comment:
Can you format this to be more readable ?

Reply:
Done